### DISCUSSION TITLE
Error after installation on multiple hosts with telemetry.js

### DISCUSSION THREAD

**Discussion by Na-Cly** [gatsbyjs/gatsby#33974] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I tried installing gatsby-cli via npm v6 on both ubuntu 20.04 and rocky linux 8 and experience the same issue. 

/usr/local/lib/node_modules/gatsby-cli/node_modules/gatsby-telemetry/lib/telemetry.js:39
  store = new _eventStorage.EventStorage();
        ^

SyntaxError: Unexpected token =
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/gatsby-cli/node_modules/gatsby-telemetry/lib/index.js:21:18)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10) 

### npm and node verison: 

[root]# node --version
v10.24.0

[root]# npm --version
6.14.11 

### command used to install: 

npm install --global gatsby-cli

### installation output:

npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/joi@15.1.1: Switch to 'npm install joi'
npm WARN deprecated @hapi/address@2.1.4: Moved to 'npm install @sideway/address'
npm WARN deprecated @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/topo@3.1.6: This version has been deprecated and is no longer supported or maintained
/usr/local/bin/gatsby -> /usr/local/lib/node_modules/gatsby-cli/cli.js

> gatsby-telemetry@3.1.2 postinstall /usr/local/lib/node_modules/gatsby-cli/node_modules/gatsby-telemetry
> node src/postinstall.js || true

> gatsby-cli@4.1.3 postinstall /usr/local/lib/node_modules/gatsby-cli
> node scripts/postinstall.js

npm WARN notsup Unsupported engine for gatsby-cli@4.1.3: wanted: {"node":">=14.15.0"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: gatsby-cli@4.1.3
npm WARN notsup Unsupported engine for fs-extra@10.0.0: wanted: {"node":">=12"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: fs-extra@10.0.0
npm WARN notsup Unsupported engine for gatsby-core-utils@3.1.2: wanted: {"node":">=14.15.0"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: gatsby-core-utils@3.1.2
npm WARN notsup Unsupported engine for gatsby-telemetry@3.1.2: wanted: {"node":">=14.15.0"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: gatsby-telemetry@3.1.2
npm WARN notsup Unsupported engine for value-or-promise@1.0.6: wanted: {"node":">=12"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: value-or-promise@1.0.6
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.2 (node_modules/gatsby-cli/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN notsup Unsupported engine for type-fest@2.5.3: wanted: {"node":">=12.20"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: type-fest@2.5.3

+ gatsby-cli@4.1.3
added 676 packages from 374 contributors in 38.223s

### Reproduction Link

N/a

### Steps to Reproduce

1. Install gatsby-cli via npm v6.14.11
2. run any gatsby command 

...

### Expected Result

be able to use gatsby

### Actual Result

``` /usr/local/lib/node_modules/gatsby-cli/node_modules/gatsby-telemetry/lib/telemetry.js:39
  store = new _eventStorage.EventStorage();
        ^

SyntaxError: Unexpected token =
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/gatsby-cli/node_modules/gatsby-telemetry/lib/index.js:21:18)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10) ```

### Environment

```shell
Ubuntu 20.04 LTS 
Rocky Linux release 8.4 (Green Obsidian)
```

**Comment by LekoArts** [comment]
Hi!

You'll need to use at least Node v14.15.0

**Reply by daltongarbin** [reply]
It worked for me. Thanks!

**Reply by dhruvv7** [reply]
Worked for me as well. In my case, Ubuntu 20.04, apt-get was getting node v10. Once updated through 
`
curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -`
`sudo apt-get install -y nodejs`

was able to go ahead with gatsby new command

**Comment by f-bog** [comment]
I am also experiencing this problem... using node v14.16.0
